src/pages/AddPersonalBankAccountPage.js

```diff
@@ -93,7 +98,10 @@ class AddPersonalBankAccountPage extends React.Component {
                         description={this.props.translate('addPersonalBankAccountPage.successMessage')}
                         shouldShowButton
                         buttonText={this.props.translate('common.continue')}
-                        onButtonPress={() => Navigation.navigate(ROUTES.SETTINGS_PAYMENTS)}
+                        onButtonPress={() => {
+                            BankAccounts.clearPersonalBankAccount();
```

Comment:
Clearing this early cause a regression https://github.com/Expensify/App/issues/15202